app/code/Magento/Store/Model/StoreManager.php

```diff
@@ -190,7 +205,11 @@ public function getStores($withDefault = false, $codeKey = false)
     }
 
     /**
-     * {@inheritdoc}
+     * Retrieve application website object
```

Comment:
Please revert the comment and just remove curly braces `{}`. Please refer to the guide https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#inheritdoc